Dalamud/Plugin/Internal/Types/PluginRepository.cs

```diff
@@ -37,10 +37,18 @@ internal class PluginRepository
         Timeout = TimeSpan.FromSeconds(20),
         DefaultRequestHeaders =
         {
+            Accept =
+            {
+                new MediaTypeWithQualityHeaderValue("application/json"),
+            },
             CacheControl = new CacheControlHeaderValue
             {
                 NoCache = true,
             },
+            UserAgent =
+            {
+                new ProductInfoHeaderValue("Dalamud", $"{Util.GetGitHash()}[{Util.GetGitCommitCount()}]"),
```

Comment:
Could you change this to use the version number (i.e. `Dalamud/9.0.0.2`) instead? imo we'd prefer that; it feels more likely to be useful as a UA, and is more convention-following/standards-abiding than a hash.

Reply:
I didn't know where to find that, but I'll search around!

Reply:
```cs
ImGui.MenuItem(dalamud.StartInfo.GameVersion?.ToString() ?? "Unknown version", false);
```
Looks like that's not accessible from a static context, though.

Reply:
> ```cs
> ImGui.MenuItem(dalamud.StartInfo.GameVersion?.ToString() ?? "Unknown version", false);
> ```
> 
> Looks like that's not accessible from a static context, though. Unless you just mean use reflection for AssemblyVersion?

er, yeah, sorry for the ambiguity - the Dalamud AssemblyVersion is what we're looking for here! iirc `Util.AssemblyVersion` exists for this